site/content/docs/development/architecture.md

```diff
@@ -111,7 +110,7 @@ When Cartographer reconciles an owner, each resource in the matching blueprint i
 ### ClusterSupplyChain
 A ClusterSupplyChain blueprint continuously integrates and builds your app.
 
-![ClusterSupplyChain](../img/supplychain.jpg)
+![ClusterSupplyChain](../img/supplychain.png)
```

Comment:
I still find this image a bit tough to read on the site- it looks fine in the repo but I can't make out a lot of the words on the site.

Reply:
Yep https://github.com/vmware-tanzu/cartographer/issues/438